udata/core/user/api.py

```diff
@@ -169,8 +169,7 @@ def get(self):
         return [
             {
                 'id': opt['text'],
-                'first_name': opt['payload']['first_name'],
-                'last_name': opt['payload']['last_name'],
+                'fullname': opt['payload']['fullname'],
```

Comment:
It's intended to be the same fields as user or user reference.
It also match the fields from ``user/search.py``.

I think this is the ``js/components/form/user-completer.vue`` which need to be fixed.